### DISCUSSION TITLE
Welcome to hugo-PaperMod Discussions!

### DISCUSSION THREAD

**Discussion by adityatelange** [adityatelange/hugo-PaperMod#118] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by parsi007** [comment]
hi

**Comment by ah969300815** [comment]
Hey this my first comment,so thanks

**Comment by afreisinger** [comment]
Hi @adityatelange, wonderfull !!
I have a question.
On the example site,  Web Server is available at http://localhost:1313/hugo-PaperMod/ .
Where you config the path ?
I want to deploy on http://localhost:1313/
Thanks

**Reply by RoneoOrg** [reply]
Try to add `baseURL = "/"` to `config.toml`
